### PR TITLE
Add Alpine Linux as supported platform

### PR DESCRIPTION
**Pull request by Watson1978**
This PR will add Alpine Linux as supported platform.

### CODE REVIEW

File: lib/native-package-installer/platform/alpine-linux.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2017-2023  Ruby-GNOME Project Team
```
Comment (kou) [inline lib/native-package-installer/platform/alpine-linux.rb:1]:
```suggestion
# Copyright (C) 2023  Ruby-GNOME Project Team
```

File: lib/native-package-installer/platform.rb [hunk 2]
```diff
@@ -46,6 +46,7 @@ def detect
 
 require_relative "platform/debian"
 require_relative "platform/ubuntu"
+require_relative "platform/alpine-linux"
```
Comment (kou) [inline lib/native-package-installer/platform.rb:49]: These `require_relative`s are for Debian based distributions.
Could you put this after `platform/suse`?

```diff
diff --git a/lib/native-package-installer/platform.rb b/lib/native-package-installer/platform.rb
index 8476fbd..528c785 100644
--- a/lib/native-package-installer/platform.rb
+++ b/lib/native-package-installer/platform.rb
@@ -57,6 +57,8 @@ require_relative "platform/amazon-linux-2023"
 
 require_relative "platform/suse"
 
+require_relative "platform/alpine-linux"
+
 
 # Windows
 require_relative "platform/msys2"
```

**Review by kou — COMMENTED** [review]
Could you also add Alpine Linux to CI?

```diff
diff --git a/.github/workflows/test.yml b/.github/workflows/test.yml
index 2bef357..711740e 100644
--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -10,6 +10,7 @@ jobs:
       matrix:
         distribution:
           - almalinux-9
+          - alpine-linux
           - alt-linux
           - amazon-linux-2
           - amazon-linux-2023
```

**Comment by Watson1978** [comment]
@kou Thank you for your review. I fixed at https://github.com/ruby-gnome/native-package-installer/pull/19/commits/451266b736c782351fa53b7990e9b1708efb49d3

**Comment by kou** [comment]
Thanks!

**Pull request merged**